src/aiy/_drivers/_status_ui.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A status UI powered by the LED on the VoiceHat."""
+
+import logging
+import os.path
+
+import aiy.audio
+
+# Location of the LED status-ui service's FIFO file.
+_LED_FIFO = "/tmp/status-led"
+
+logging.basicConfig(
```

Comment:
Remove this logging config. The logging config should be set up in the main() function (or similar). Otherwise, users who import this will have their logging unexpectedly reconfigured.